kornia/augmentation/container/augment.py

```diff
@@ -158,7 +158,8 @@ def apply_by_key(
         if param is not None:
             assert module_name == param.name
         if module is None:
-            module = self.get_submodule(module_name)
+            # TODO (jian): double check why typing is crashing
```

Comment:
@shijianjian please check this later. Seems that with some python version this is crashing